adapters/RPGRegionsAdapter/src/main/kotlin/com/caleb/typewriter/rpgregions/entries/event/EnterRegionEventEntry.kt

```diff
@@ -0,0 +1,33 @@
+package com.caleb.typewriter.rpgregions.entries.event
+
+import net.islandearth.rpgregions.api.events.RegionsEnterEvent
+import lirand.api.extensions.server.server
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.*
+import me.gabber235.typewriter.entry.entries.EventEntry
+import me.gabber235.typewriter.utils.Icons
+import java.util.*
+
+
+@Entry("on_enter_RPGRegions", "When a player enters a RPGRegions region", Colors.YELLOW, Icons.SQUARE_CHECK)
```

Comment:
Please only use `snake_case` for entry names. Change this to `on_enter_rpg_region`.

Also Square Check feels like a weird Icon for this. Have a look at [font awesome icons](https://fontawesome.com/search?o=r&m=free) to see the available icons (only the free ones).